updatersync/src/main/java/nl/openweb/hippo/groovy/watch/GroovyFilesWatcher.java

```diff
@@ -64,10 +63,10 @@ private FileSystemObserver observeFileSystemIfNeeded() {
         if (projectBaseDir == null) {
             return null;
         }
-        if (config.getWatchedModules().size() > 0) {
+        if (!config.getWatchedModules().isEmpty()) {
```

Comment:
Readability, you don't have to invert this instead just swap the if else itself. reading the code stay's more easy that way

Reply:
done